parsec/core/backend_connection/cmds.py

```diff
@@ -679,6 +681,8 @@ async def organization_bootstrap(
         root_verify_key=root_verify_key,
         user_certificate=user_certificate,
         device_certificate=device_certificate,
+        redacted_user_certificate=redacted_user_certificate,
+        redacted_device_certificate=redacted_device_certificate,
```

Comment:
Funny we missed that!

Reply:
It's because we didn't used those fields until now ! (see next comment)